dietpi/dietpi-software

```diff
@@ -8966,6 +9010,43 @@ _EOF_
 			pihole -w freegeoip.app --comment 'Used by dietpi-banner and dietpi-vpn to obtain your public IP and location'
 
 		fi
+		
+		software_id=126 # AdGuard Home
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			
+			if [[ ! -f '/opt/AdGuardHome/AdGuardHome.yaml' ]]; then
+				dps_index=$software_id Download_Install 'AdGuardHome.yaml' /opt/AdGuardHome/AdGuardHome.yaml
+				G_CONFIG_INJECT 'password:[[:blank:]]' "  password: $(htpasswd -bnBC 10 "" $GLOBAL_PW | tr -d ':\n' | sed 's/$2y/$2a/')" /opt/AdGuardHome/AdGuardHome.yaml
+			fi
+
+			# Service
+			cat << _EOF_ > /etc/systemd/system/AdGuardHome.service
```

Comment:
Let's not use capital letters for the service name. Probably it works, but it is very uncommon.

I would vote for this for all directories as well. We have a few other cases where capital letters are used for install directories, but it makes console navigation nasty (tab-completion by default is case-sensitive). Generally I like the consistency where service user = service name = install dir = data dir, all in lower-case letters 😃.

Reply:
I know, I just tried to follow the official service naming as much as possible. ;)

Reply:
ok changed to `adguardhome.service`